pgmpy/models/LinearGaussianBayesianNetwork.py

```diff
@@ -432,3 +432,38 @@ def is_imap(self, JPD):
         raise NotImplementedError(
             "is_imap method has not been implemented for LinearGaussianBayesianNetwork."
         )
+
+    def get_random(
+        n_nodes=5, edge_prob=0.5, node_names=None, n_states=None, latents=False
+    ):
+        """
+        Returns a randomly generated Linear Gaussian Bayesian Network on `n_nodes` variables
+        with edge probabiliy of `edge_prob` between variables.
+
+        Parameters
+        ----------
+        n_nodes: int
+            The number of nodes in the randomly generated DAG.
+
+        edge_prob: float
+            The probability of edge between any two nodes in the topologically
+            sorted DAG.
+
+        node_names: list (default: None)
+            A list of variables names to use in the random graph.
+            If None, the node names are integer values starting from 0.
+
+        n_states: int or dict (default: None)
+            The number of states of each variable in the form
+            {variable: no_of_states}. If a single value is provided,
+            all nodes will have the same number of states. When None
+            randomly generates the number of states.
+
+        latents: bool (default: False)
+            If True, also creates latent variables.
+
+        """
+        base_bn = BayesianNetwork.get_random(
```

Comment:
You would need to make a few changes here. Both Bayesian Network and LinearGaussianBayesianNetwork has two parts: 1) The model structure represented using the DAG class, 2) Parameterization (represented using `TabularCPD` for `BayesianNetwork` and `LinearGaussianCPD` for `LinearGaussianBayesianNetwork`).

When you call `BayesianNetwork.get_random`, it generates the model structure by calling the `DAG.get_random` method and then adds randomly generated `TabularCPD` instances to it. We need to do something similar for the `get_random` method here but instead of adding `TabularCPD`s, we need to add instances of `LinearGaussianCPD`. So, the main steps would be:

1. Generate the model structure using `DAG.get_random`.
2. Use the `LinearGaussianBayesianNetwork.get_random_cpds` method to generate random `LinearGaussianCPD`. This would also mean that the arguments required by `LinearGaussianBayesianNetwork.get_random` would be slightly different than `BayesianNetwork.get_random` as `LinearGaussianCPD`s require different arguments. 

I would suggest looking at the `BayesianNetwork.get_random` method to get an example of how everything should work.